### PR TITLE
Feature/v1.9 v4

### PR DESCRIPTION
**Pull request by whaeck**
A few simple updates:
- restricted python properties based on choice and vector occurrence
- some changes in python bindings namespaces that interfered with proper namespace resolution for types (because the python wrapper functions were in namespaces similar to the component namespaces the compiler could not properly resolve component types)
- all python cpp files are now enabled (no more paying attention to commenting out containers and transport wrapper functions)

In short: the auto-generated python bindings now all compile!

Caveat: not all C++ and python unit tests are done but at least it compiles properly, and all C++ and python unit tests run properly

### CODE REVIEW

File: autogen/json2class.cpp [hunk 1]
```diff
@@ -1671,18 +1671,18 @@ void file_python_namespace(
    cpp << "   // create the " << nsname << " submodule\n";
    cpp << "   python::module submodule = module.def_submodule(\n";
    cpp << "      \"" << nsname << "\",\n";
-   cpp << "      \"GNDS v1.9 " << nsname << "\"\n";
+   cpp << "      \"GNDS " << Version << " " << nsname << "\"\n";
```
Comment (staleyLANL) [inline autogen/json2class.cpp:1674]: I can't believe I missed that accidental hardcoding. :-/

Comment (whaeck) [reply autogen/json2class.cpp:1674]: I just noticed that by chance.

**Review by staleyLANL — APPROVED** [review]
Look good. I'll trust all the python changes. Approving now...
